lib/netext/grpcext/conn.go

```diff
@@ -162,8 +162,11 @@ func (c *Conn) Invoke(
 		// {"x":6,"y":4,"z":0}
 		raw, _ := marshaler.Marshal(resp)
 		msg := make(map[string]interface{})
```

Comment:
Okay, I've looked into this and believe that this line is an issue, switching to this should help
```suggestion
		var msg interface{}
```